.github/workflows/test.yaml

```diff
@@ -21,3 +21,43 @@ jobs:
               run: |
                   docker compose -f docker-compose.integration.yml down -v &&
                   docker compose -f docker-compose.integration.yml up --build --force-recreate --abort-on-container-exit --exit-code-from app
+
+
+    prettier:
+            name: Format code with Prettier
+            runs-on: ubuntu-latest
+            steps:
```

Comment:
btw, what was the reason for extracting this as a separated workflow job and not included in `ci`na?